apple/testing/default_runner/ios_test_runner.template.sh

```diff
@@ -146,17 +146,19 @@ if [[ -n "${command_line_args}" ]]; then
   LAUNCH_OPTIONS_JSON_STR+="\"args\":[\"$command_line_args\"]"
 fi
 
+TEST_FILTER="%(test_filter)s"
+
 # Use the TESTBRIDGE_TEST_ONLY environment variable set by Bazel's --test_filter
 # flag to set tests_to_run value in ios_test_runner's launch_options.
 # Any test prefixed with '-' will be passed to "skip_tests". Otherwise the tests
 # is passed to "tests_to_run"
-if [[ -n "$TESTBRIDGE_TEST_ONLY" ]]; then
+if [[ -n "$TESTBRIDGE_TEST_ONLY" || -n "$TEST_FILTER"]]; then
   if [[ -n "${LAUNCH_OPTIONS_JSON_STR}" ]]; then
     LAUNCH_OPTIONS_JSON_STR+=","
   fi
 
   IFS=","
-  ALL_TESTS=("$TESTBRIDGE_TEST_ONLY")
+  ALL_TESTS=("$TESTBRIDGE_TEST_ONLY,$TEST_FILTER")
```

Comment:
if only TESTBRIDGE_TEST_ONLY or only TEST_FILTER are set I think this would leave you with an unexpected `,`?

Reply:
Yeah likely, wasn't effecting my tests so didn't notice. 

I added a different way, through checking the different combinations. There is likely a more elegant solution than how I did it but not a huge bash aficionado and trying to use `+=` on different parts wasn't working as expected so ended up with this. If you know of another way please let me know